### ISSUE TITLE
View an order for a customer

### ISSUE THREAD

**Issue by divyanshusharma1709** [CSCI-GA-2820-SU24-001/orders#3]
**As a** customer/orders manager
**I need** to view an order
**So that** I can view the items and their quantities that I have ordered
   
 ### Details and Assumptions
 * The customer ID and the order ID is provided
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given a customer with orders and a specific order 
 When a GET request is made with the customer ID and the order ID
 Then return the order details in JSON format with a 200 status code. Details to be returned include:
 1. Items in the order
 2. The quantity of each item in the order
 3. Individual prices of each item
 4. The overall cost of the order
 5. Date the order was placed
```

**Comment by ziyuzoo** [comment]
I think for MVP let's take out shipping address / delivery date? since there's nothing for it in the other request shapes and we don't have another team's API to get this data from

**Comment by yl12065** [comment]
If we consider the orders manager can delete and update the order, should he/she also be able to view the orders?

**Comment by divyanshusharma1709** [comment]
@ziyuzoo I think that makes sense. Updated.
@yl12065 Yes, you are right. Added to the issue. Thanks for the feedback!

**Comment by divyanshusharma1709** [comment]
Closed by merging #19
